src/App.tsx

```diff
@@ -42,20 +42,19 @@ import {CLOUD} from 'src/shared/constants'
 import {executeVWO} from 'src/utils/vwo'
 
 // Styles
-import './MultiOrgOverrideStyles.scss'
+import 'src/identity/styles/MultiOrgStyles.scss'
```

Comment:
i think this actually belongs in `chronograf.scss` now since it's used app-wide

Reply:
💯 . I'll move this into the chronograf scss file.